lib/src/base-styles/_nx-tables.scss

```diff
@@ -365,29 +365,89 @@
   }
 }
 
-.nx-cell__sort-btn {
-  height: nx-table-variables.$table-cell-header-height;
+.nx-cell {
+  .nx-cell__sort-btn {
+    height: nx-table-variables.$table-cell-header-height;
+    position: relative;
+  
+    > * {
+      position: relative;
+    }
+  
+    &::before {
+      box-sizing: border-box;
+      border-radius: 4px;
+      content: '';
+      // offset by 1px on top and 2px on the bottom
+      height: calc(100% - 3px);
+      position: absolute;
+      left: 0;
+      top: 1px;
+      width: calc(100% - 1px);
+    }
+  
+    &:hover {
+      &::before {
+        background-color: var(--nx-color-table-header-button-background-hover);
+      }
+    }
+    &:focus {
+      &::before {
+        border: var(--nx-focus-outline-width) solid var(--nx-color-interactive-border-focus);
+      }
+    }
+    &:active {
+      &::before {
+        background-color: var(--nx-color-table-header-button-background-active);
+        border: none;
+      }
+    }
+  }
+  &:first-child {
+    .nx-cell__sort-btn {
+      &::before {
+        left: 1px;
```

Comment:
Shouldn't this apply to all of them? I assume the hover visual should be inset 1px on the top, left, and right not matter which column it's in

Reply:
> Shouldn't this apply to all of them? I assume the hover visual should be inset 1px on the top, left, and right (and 2px from the bottom as you have) no matter which column it's in

Yep, I confirmed this with Jason, my mistake. [f488f71](https://github.com/sonatype/sonatype-react-shared-components/pull/1156/commits/f488f71b61f1f4d8dad07cb12bc5981bbeead1eb)